lib/accounts.js

```diff
@@ -1,45 +1,45 @@
 const got = require('got');
 const _ = require('lodash');
 
-module.exports = function(zuoraClient) {
+module.exports = function (zuoraClient) {
   function getAccount(accountId) {
-    return zuoraClient.authenticate().then(headers => {
+    return zuoraClient.authenticate().then((headers) => {
       const url = zuoraClient.serverUrl + '/v1/object/account/' + accountId;
       const query = {
         headers,
-        json: true
+        json: true,
       };
-      return got.get(url, query).then(res => res.body);
+      return got.get(url, query).then((res) => res.body);
     });
   }
 
   function deleteAccount(accountId) {
-    return zuoraClient.authenticate().then(headers => {
+    return zuoraClient.authenticate().then((headers) => {
       const url = zuoraClient.serverUrl + '/v1/object/account/' + accountId;
       const query = {
         headers,
-        json: true
+        json: true,
       };
-      return got.delete(url, query).then(res => res.body);
+      return got.delete(url, query).then((res) => res.body);
     });
   }
 
   function updateAccount(accountId, updatedContent) {
-    return zuoraClient.authenticate().then(headers => {
-      const url = zuoraClient.serverUrl + '/v1/object/account/' + accountId;
+    return zuoraClient.authenticate().then((headers) => {
+      const url = zuoraClient.serverUrl + '/v1/accounts/' + accountId;
```

Comment:
Didn't have in mind the we were exposing the `/object/account` Zuora API through here instead of the newest one.
I think making this change would be a breaking change, as we use some fields that are not updatable through the new route (for instance 'Status' field).
I'd suggest adding exporting this API with a different name and keeping both, so we can use one or the other depending on our need, WDYT ?

Reply:
Agreed, if these two don't provide access to all of the same fields, then this is better to create a new endpoint.  I've updated the code to provide an updateAccountByKey endpoint.